app/views/links/index.html.erb

```diff
@@ -1,14 +1,13 @@
 <section class="wrapper">
   <div class="row mt">
-    <div class="col-lg-1 col-md-1 ">
-      <%= link_to 'All', links_path, 'data-no-turbolink' => true %>
-      <%= link_to 'Favourites', favourite_links_path, 'data-no-turbolink' => true %>
-    </div>
-    <div class="col-lg-8 col-md-8 col-sm-12 links" id="links-list">
-     <%= render partial: 'links/links_list', locals: { links: @links}%>
-  </div><!--/ row -->
-    <%= will_paginate @links %>
-    </div>
-</section>
-
-
+    <% if @links.blank? %>
+      <div class="no_links_msg">You do not have anything to display at this time. <%= link_to 'Start saving one now!', new_link_path%></div>
+    <% else %>
+      <%= render 'links/left_sidebar' %>
```

Comment:
As commented earlier, sidebar should be visible all the time so it should not be part of links only but to all.